assets/ios/swift/CpdDark.swift

```diff
@@ -160,6 +160,35 @@ public class CpdDarkDesignTokens {
     public static let cpdColorGray300 = UIColor(red: 0.886, green: 0.902, blue: 0.918, alpha: 1)
     public static let cpdColorGray200 = UIColor(red: 0.933, green: 0.945, blue: 0.953, alpha: 1)
     public static let cpdColorGray100 = UIColor(red: 0.969, green: 0.973, blue: 0.976, alpha: 1)
+    public static let cpdFontLetterSpacingHeadingXl = 1.2%
+    public static let cpdFontLetterSpacingHeadingLg = 1.4%
+    public static let cpdFontLetterSpacingHeadingMd = -1.2%
+    public static let cpdFontLetterSpacingHeadingSm = -2.3%
+    public static let cpdFontLetterSpacingBodyLg = -2.6%
+    public static let cpdFontLetterSpacingBodyMd = -1.6%
+    public static let cpdFontLetterSpacingBodySm = -0.6%
+    public static let cpdFontLetterSpacingBodyXs = 0%
+    public static let cpdFontSizeHeadingXl = 34
+    public static let cpdFontSizeHeadingLg = 28
+    public static let cpdFontSizeHeadingMd = 22
+    public static let cpdFontSizeHeadingSm = 20
+    public static let cpdFontSizeBodyLg = 17
+    public static let cpdFontSizeBodyMd = 15
+    public static let cpdFontSizeBodySm = 13
+    public static let cpdFontSizeBodyXs = 12
+    public static let cpdFontLineHeightHeadingXlRegular = 41
+    public static let cpdFontLineHeightHeadingLgRegular = 34
+    public static let cpdFontLineHeightHeadingMdRegular = 28
+    public static let cpdFontLineHeightHeadingSmRegular = 25
+    public static let cpdFontLineHeightBodyLgRegular = 22
+    public static let cpdFontLineHeightBodyMdRegular = 20
+    public static let cpdFontLineHeightBodySmRegular = 18
+    public static let cpdFontLineHeightBodyXsRegular = 16
+    public static let cpdFontWeightBold = 700
+    public static let cpdFontWeightSemibold = 600
+    public static let cpdFontWeightRegular = 400
+    public static let cpdFontFamilyMono = SF Mono
```

Comment:
Does not look like this is going to compile

Reply:
@gsouquet We basically need to define how do we want to provide tokens to iOS platform. I don't think atomic typography tokens like letter spacing, font size, and such are going to be useful for their needs (that's why I added _"Most useful for generating Figma Styles."_, as it's the immediate priority).

Maybe the just need to know how our composite tokens map to built-in _text styles_, and maybe even when to combine them with system-provided modifiers (aka _symbolic traits_) like bold or tight leading (if we add support for it).

https://developer.apple.com/design/human-interface-guidelines/foundations/typography/#using-system-fonts

Reply:
@gsouquet Actually, that's also why I initially labelled atomic typography tokens for iOS as N/A in the [internal doc on Typography tokens](https://github.com/vector-im/compound-internal/blob/internal-docs/docs/typography.md#ios) transformations.

Reply:
sorry that was a very unhelpful comment from my side. It was more a note to myself to review this.

I've schedule a call with doug tomorrow to go through all of that

Reply:
Awesome. 👍

Reply:
I'd love to join the call, if possible, but it's going to be difficult tomorrow (already quite a lot of calls scheduled), so please let me know of any outcomes/questions afterward.